test/integration/services/bookstore/integration_test.go

```diff
@@ -0,0 +1,34 @@
+package bookstore
+
+import (
+	"flag"
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-service-operator-k8s/test/integration/services"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+func init() {
+	services.RegisterFlags(flag.CommandLine)
+}
+
+func TestIntegration(t *testing.T) {
+	flag.Parse()
+	framework.AfterReadingAllFlags(&framework.TestContext)
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Amazon Service Operator K8s BookStore")
+}
+
+var _ = BeforeSuite(func() {
+	fmt.Printf("Using KUBECONFIG=\"%s\"\n", framework.TestContext.KubeConfig)
+})
+
+var _ = Describe("[bookstore-integration]", func() {
+	It("should able to run", func() {
+		fmt.Println("Tests running successfully")
```

Comment:
@varun1524, is it missing test case here ? may be add a //TODO comment here

Reply:
that is the next thing anyway we are working on so not needed and as per offline discussion resolving it.